.travis.yml

```diff
@@ -102,10 +113,16 @@ script:
   - make && sudo env PATH=$PATH make install && awesome --version
   - |
     if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
-      if [ "$DO_COVERAGE" = "true" ]; then
+      do_codecov samples
+    fi
+  - |
+    if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
+      if [ "$DO_COVERAGE" != "0" ]; then
```

Comment:
Well, the 0 value is needed by cmake, AFAIK. I didn't add the `export DO_COVERAGE=0`, it was already there before (needed for the code for `tests/samples` in CMake).